syntax = "proto3";

message RegistrationRequest{
    string username = 1;
    string password = 2;
    string confrim_password  = 3;
}

message RegistrationResponse{
    bool success = 1;
    string message = 2;
}

message LoginRequest{
    string username = 1;
    string password = 2;
}

message LoginResponse{
    bool success = 1;
    string message = 2;
}

service AuthService{
    rpc Register(RegistrationRequest) returns (RegistrationResponse);
    rpc Login(LoginRequest) returns (LoginResponse);
}